index.js

```diff
@@ -58,7 +58,7 @@ betterLog.install = function (newConfig) {
 };
 
 betterLog.uninstall = function () {
-	return console.log = log;
+	return (console.error = error) && (console.log = log);

```

Comment:
Better to do just

```javascript
console.error(error);
return console.log = log;
```

Otherwise in environment that supports `console.log` but doesn't support `console.error` (there is in fact many of those), `log` won't be restored.